sacad/sources/coverartarchive.py

```diff
@@ -0,0 +1,57 @@
+"""Cover art archive cover source."""
+
+from sacad import __version__
+from sacad.cover import CoverImageMetadata, CoverSourceResult, CoverSourceQuality
+from sacad.sources.base import CoverSource
+import aiohttp
+import json
+
+
+class CoverArtArchiveSource(CoverSource):
+    """
+    Cover source using the Cover Art Archive official API.
+
+    The documentation of the API is available at
+    https://musicbrainz.org/doc/Cover_Art_Archive/API
+
+    """
+
+    # The cover art archive is tied with Musicbrainz.
+    # Therefore, in order to get a cover, one must first get a release
+    # identifier or release group identifier.
+    # To get the release (or release group) id, one can either read from
+    # the tags (which isn't allowed for now in sacad), or one can ask the
+    # musicbrainz database for identifiers from the album and artist name.
+    # See https://musicbrainz.org/doc/MusicBrainz_API for the API of
+    # musicbrainz.
+    BASE_URL = "https://musicbrainz.org/ws/2"
+    API_KEY = ""
+    API_SECRET = ""
+
+    def getSearchUrl(self, album, artist):
+        return CoverSource.assembleUrl(
+            f"{self.BASE_URL}/release-group", {"query": f"artist:{artist} AND album:{album}", "fmt": "json"}
+        )
+
+    def updateHttpHeaders(self, headers):
+        headers["User-Agent"] = f"sacad/{__version__}"
```

Comment:
I don't think this should be here because this is not specific to this source.

I've just added the same thing in the base class on the master branch.

Reply:
Rebase on master branch, you will get commit https://github.com/desbma/sacad/commit/6f7d66f3a7a2b5d1e9a593979379fd1a5f042025 which does the same thing by default and makes this unnecessary.